crates/astria-sequencer/src/accounts/action.rs

```diff
@@ -85,6 +86,15 @@ impl ActionHandler for TransferAction {
         state: &S,
         from: Address,
     ) -> Result<()> {
+        ensure!(
+            state
+                .get_bridge_account_rollup_id(&from)
+                .await
+                .context("failed to get bridge account rollup id")?
+                .is_none(),
```

Comment:
This logic seems funky. What happens in `get_bridge_account_rollup_id` if the account is not a bridge account? Is that just an `Ok(None)`? The context here makes it sounds like we failed to find a bridge account  rollup for the address, and then we want there to be none.

Also why use the get rollup id vs just checking for the bridge account existence?

Reply:
the context is just checking if `get_bridge_account_rollup_id` returned an error. if no rollup id is found for the bridge account, that means the bridge account does not exist. there's no explicit method for checking "existence" other than this